data/moves.ts

```diff
@@ -5901,6 +5901,9 @@ export const Moves: {[moveid: string]: MoveData} = {
 				this.add('-endability', pokemon);
 				this.singleEvent('End', pokemon.getAbility(), pokemon.abilityData, pokemon, pokemon, 'gastroacid');
 			},
+			onCopy(pokemon) {
+				if (this.dex.getAbility(pokemon.ability).isPermanent) pokemon.removeVolatile('gastroacid');
```

Comment:
Actually that code is out of date and should probably say
```suggestion
				if (pokemon.getAbility().isPermanent) pokemon.removeVolatile('gastroacid');
```